Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -588,6 +744,69 @@ def create_body_create_rule(rule_type: str, address: List, number: int,
         }
 
 
+def create_body_create_rule_for_v10(rule_type: str, address: List, number: int,
+                                    from_to_list: list[dict[str, Any | None]], state: str = "Enabled") -> tuple:
+    """ create part of the body for the command create/update_rule_object for v10
+        Args:
+            rule_type: str - The type of the rule.
+            address: List - A list of addresses, if relevant.
+            number: int - The number of the IPV.
+            from_to_list: List - A list that contains dictionaries with from and do addresses.
+            stste: str - An Enabled or Disabled state.
+        Returns:
+            Returns the body for the request.
+        """
+    # build a list of dictionaries with the state and the address
+    list_to_send: list[dict] = [
+        {"value": single_address, "state": STATE_TO_NUMBER.get(state)}
+        for single_address in address]
+    # for parameters with a range, we need to add the state to the dictionary
+    if from_to_list:
+        from_to_list[0].update({"state": STATE_TO_NUMBER.get(state)})
+
+    if 'HOST' in rule_type:
+        return f'HostIPv{number}', {
+            f'hostIPv{number}AddressList': list_to_send
+        }
+    elif 'ADDRESS_RANGE' in rule_type:
+        return f'IPv{number}AddressRange', {
+            f'IPV{number}RangeList': from_to_list
+        }
+    else:
+        return f'Network_IPV_{number}', {
+            f'networkIPV{number}List': list_to_send
+        }
+
+
+def modify_v10_results_to_v9_format(results: List[Dict[Any, Any]]) -> List[Dict[Any, Any]]:
+    # modify the results of v10 to be in the same format as in v9.
+    # in the nested dictionary,  the key is the name of the object,
+    # and the value instead of being a list of dictionaries, now the value is a list of strings
```

Comment:
```suggestion
    # Modify the results (API response) of v10 to v9 format.
    Modifications In the nested dictionary:
    - The key is the name of the object.
    - The value is a list of strings instead of a list of dictionaries.
```
Also, let's add it as a docstring and not as comments.